### ISSUE TITLE
Pretrained model for student network

### ISSUE THREAD

**Issue by he-y** [yuanli2333/Teacher-free-Knowledge-Distillation#15]
Thanks for the great work.

I find the pre-trained model for the teacher network.
Will you release the Pretrained model for student network?

Thanks!

**Comment by yuanli2333** [comment]
Hi,
There are many kinds of student network in our work. The teacher model can also be student (reverse-KD), and the model itself can also be student (self-teaching).  So which pretrained models do you mean?
I think the simplestest way to get the model parameters is to train by youself based on our codes. It is simple and fast to run our codes.

**Comment by he-y** [comment]
Thanks for the reply.
Is it possible to release the student model based on the accuracy table? This might be easier for us to use.

For example, for every accuracy (such as 71.05) in the table below, a model corresponds to the setting could be released.
![image](https://user-images.githubusercontent.com/10541284/86986092-25d37a80-c1d6-11ea-926c-9c82dca47f0e.png)

This will save lots of time for rerunning the experiments.
Thank you very much!
